packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -258,10 +292,13 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
       contextMenuLocation: _contextMenuLocation,
       previousChildRect: _decoyChildEndRect!,
       builder: (BuildContext context, Animation<double> animation) {
-        if (widget.previewBuilder == null) {
-          return widget.child;
+        if(widget.builder != null) {
+          final AnimationController controller = AnimationController(duration: const Duration(seconds: 0), vsync: this); 
+          final localAnimation = Tween<double>(begin: 0, end: 1).animate(controller);
+          controller.forward();
+          return widget.previewBuilder!(context, animation, widget.builder!(context, localAnimation));
         }
```

Comment:
@justinmc this is how I currently am passing along the builder for the open context menu, if the developer provides a builder. I was originally just passing along the builder, but that caused the developer defined animation to play twice, so I created an animation instance, and completed it, in order to get the final state of the widget from the builder to pass along to the `_ContextMenuRoute`. Is this ok, or is there a better way to do this?

Reply:
Ping @justinmc if you missed it, but double checking that this is an ok way to get a static widget from a builder with an animation? Not sure if this is the "correct" way, or if there's a better one.

Reply:
Ah sorry.  In the [proposal](https://github.com/flutter/flutter/pull/110616#pullrequestreview-1151602906) that I commented with most recently, I think this would be much simpler.  The builder would be non-nullable and we would use it no matter what, without needing to mess with the animation.  If I'm not missing anything.

```dart
  builder: (BuildContext context, Animation<double> animation) {
    return widget.builder(context, animation);
  },
```

Reply:
Oh, ok. I see what you're saying. 👍